### DISCUSSION TITLE
VBScript issue

### DISCUSSION THREAD

**Discussion by Narasimhara** [ClearFoundry/ClearScript#306] [General]
hi All,

Could you please help me out how to call this Main function

SUB Main () 
    MsgBox "End of Script", vbInformation
	
End SUB

Iam not getting message box after the Main function below is the code:

            engine = new VBScriptEngine();
            engine.DocumentSettings.AccessFlags = DocumentAccessFlags.EnableAllLoading;
            engine.HostWindow = new CVWindow();
             engine.EvaluateDocument(FunctionName); // I want to send function name and call as i have very big functions.

Please help me out how to call main and execute  the statement dynamically.

**Comment by ClearScriptLib** [comment]
Hi @Narasimhara,

Please clarify. What is `FunctionName`? If it is the name of a function, why are you passing it to `EvaluateDocument`?

Thanks!
